cat-launcher/src/pages/PlayPage/ReleaseSelector.tsx

```diff
@@ -73,6 +75,29 @@ export default function ReleaseSelector({
     queryFn: () => getLastPlayedVersion(variant),
   });
 
+  useQuery({
+    queryKey: queryKeys.installationStatus(variant, selectedReleaseId),
+    queryFn: () => {
+      if (!selectedReleaseId) {
+        return;
+      }
+
+      return getInstallationStatus(variant, selectedReleaseId);
+    },
+    enabled: !!selectedReleaseId,
+    onSuccess: (status) => {
```

Comment:
In the latest version of Tanstack Query, the useQuery hook no longer has a onSuccess callback. Update the code to use the new recommended approach.